File: BAEKJOON/1436.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : __dirname + '/input.txt';
const input = Number(fs.readFileSync(filePath).toString().trim());

function solution(movieCount) {
  let seriesNum = 666;
  let nameCount = 1;

  while (nameCount < movieCount) {
    seriesNum += 1;
    if (String(seriesNum).includes('666')) nameCount++;
  }
  console.log(seriesNum);
}

solution(input);

File: baekjoon/1436.py
n = int(input())
seriesCnt, movieNum = 1, 666 # 시리즈 카운트, 영화 제목에 들어갈 수

while seriesCnt < n: # n번째 영화가 되면 탐색 종료
    movieNum += 1 # 영화 제목에 들어갈 수를 1씩 늘려가면서 '666'이 있는지 확인한다. - 완전탐색
    if '666' in str(movieNum): seriesCnt += 1

print(movieNum)